web/meetup-facilitator/src/App.jsx

```diff
@@ -16,6 +17,8 @@ function App() {
       <Router>
         <Routes>
           <Route path='/' element={<Home/>} />
+          <Route path='/activities' element={<Activities/>} />
+          <Route path='/recommendation' element={<Recommendation/>}/>
```

Comment:
Will need to make it `/recommendation:group_id` like the others so we can pass through `useParams()`.

Reply:
Additionally let's make it plural `recommendations` to be consistent with the other pages. Together `/recommendations/:group_id`